dftimewolf/lib/exporters/timesketch.py

```diff
@@ -45,11 +47,11 @@ def __init__(self,
     self._analyzers = []  # type: List[str]
     self.wait_for_timelines = False
     self.host_url = None
-    self.sketch = None  #type: ts_sketch
+    self.sketch = None
```

Comment:
Why are we removing this type? Would be good to add it, should it be/have been ts_sketch.Sketch?

Reply:
Added back.